### DISCUSSION TITLE
Release Status 4.10

### DISCUSSION THREAD

**Discussion by NeatGreat** [prisma/prisma#17895] [Q&A] [answered]
Hi, 
what is the plan regarding release 4.10 as this is already delayed by multiple days?

**Comment by nurul3101** [comment, marked as answer]
Hello @NeatGreat 👋 

Prisma version [4.10.0](https://github.com/prisma/prisma/releases/tag/4.10.0) was released last Tuesday (7th Feb) and there is also a patch release [4.10.1](https://github.com/prisma/prisma/releases/tag/4.10.1) that was released on last Friday (10th Feb).

Are you running into any issues in using these versions?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
